docs/content/en/docs/clustermgmt/security/manually-renew-certs.md

```diff
@@ -168,3 +168,74 @@ ${IMAGE_ID} tmp-cert-renew \
 5. Repeat the above steps for all control plane nodes.
 
 You can similarly use the above steps to rotate a single certificate instead of all certificates.
+
+#### Update kubeconfig on Admin machine
```

Comment:
```suggestion
#### Update your kubeconfig certificate
```

Admin machines are ephemeral. Suggesting they need to update the admin machine is a little misleading as it may no longer exist.